client/src/pages/HomePage/components/AddAnimal.js

```diff
@@ -0,0 +1,55 @@
+import React, { useState } from 'react';
+import { Grid, Typography, Button } from '@mui/material';
+import {
+  addAnimalButton,
+  addAnimalContainer,
+  addAnimalInputContainer,
+} from '../../../styles/home/addAnimalStyles';
+import AddAnimalInput from './AddAnimalInput';
+
+const AddAnimal = (props) => {
+  const { addAnimal } = props;
```

Comment:
so kay i change man ang prop name to `onAddAnimal` weird naman siya, so pwede raman na nimo ma `onAddAnimal: handleAddAnimal` boom!

Reply:
resolved